java/src/main/kotlin/org/digma/intellij/plugin/idea/psi/java/JavaPsiUtils.kt

```diff
@@ -42,14 +43,6 @@ class JavaPsiUtils {
             }
         }
 
-        @JvmStatic
-        @NotNull
-        fun getClassesWithin(psiFile: PsiFile): List<PsiClass> {
-            //it must be a PsiJavaFile so casting should be ok
-            val psiJavaFile = psiFile as PsiJavaFile
-            return psiJavaFile.classes.asList()
-        }
```

Comment:
if must be PsiJavaFile then the argument can be PsiJavaFile . this method will fails if sent another file type.

Reply:
this method been removed :-)

Reply:
ok, sorry